src-ui/src/app/components/common/filterable-dropdown/filterable-dropdown.component.ts

```diff
@@ -43,6 +45,10 @@ export class FilterableDropdownSelectionModel {
     return this.items.filter(i => this.temporarySelectionStates.get(i.id) == ToggleableItemState.Selected)
   }
 
+  getExcludedItems() {
+    return this.items.filter(i => this.temporarySelectionStates.get(i.id) == ToggleableItemState.Excluded)
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/jonaswinkler/paperless-ng/pullRequest?prid=7592508)
```suggestion
    return this.items.filter((i) => this.temporarySelectionStates.get(i.id) == ToggleableItemState.Excluded)
```